db/migrate/20220217104803_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Post do |t|
+      t.references :User, null: false, foriegn_key: true, index:true
```

Comment:
- [ ] <b>REQUIRED:</b> For `user_id` foreign key, the project requires to use `author_id` instead. Here is how you can anchieve that
```rb
t.references :author, foreign_key: { to_table: 'users' }